spec/pb/thing_controller.rb

```diff
@@ -15,65 +15,68 @@
 # OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #
 require 'rpc/ThingService_services_pb'
+require 'rpc/Error_pb'
 require 'ffaker'
 
-class ThingService < ::Rpc::ThingService::Service
-  include Gruf::Service
+class ThingController < ::Gruf::Controllers::Base
```

Comment:
Can we provide a more concrete example in the specs and documents? I feel like `Thing*` is just too abstract to easily comprehend. I'd like to see us use `User*`, as most developers can relate to that or have built systems with users. While the code will be the same, I think it will be easier to apply to the developer's situation using a less abstract concept.

Reply:
Pushing to gruf 2.1